baseplate/integration/thrift/command.py

```diff
@@ -55,6 +55,20 @@ def run(self):
 
                 self.copy_tree(input_package, output_package)
 
+                # rewrite "from thriftname" to "from package.thriftname_thrift"
```

Comment:
i really really don't love this. the generated "remote" script (simple client for local CLI testing) uses absolute imports but because of our shenanigans and the fact that we nest inside the application package, those references are incorrect. this is a horrid hack to try and rewrite the import. :see_no_evil:

Reply:
is the remote script the thing generated above?

Reply:
the above is generating all the thrift code which does include the remote script yeah